scripts/jenkins/ardana/ansible/roles/input_model_generator/vars/templates/network/compact.yml

```diff
@@ -24,13 +31,6 @@ network_groups:
     routes:
       - default
 
-  - name: MANAGEMENT
```

Comment:
It looks like this block simply moved, is that deliberate? Maybe we can shrink the diff by leaving this block unchanged?

Reply:
Thanks for your review Ryan!
This template is being used to generate  network model. So, order is important to have MGMT network is being created with ipv4 cidr in mix-mode deployment. This also helps supporting following logic when generating networks.yml for mix-mode.

{%   if ipv6 == true %}
{%   if ns.net_id != 100 %}
      cidr: fc00:0:0:{{ ns.net_id }}::/64
      gateway-ip: fc00:0:0:{{ ns.net_id }}::1
{%   else %}
      cidr: 192.168.{{ ns.net_id }}.0/24
      gateway-ip: 192.168.{{ ns.net_id }}.1